static/js/HeaderAutocomplete.jsx

```diff
@@ -362,7 +362,8 @@ export const HeaderAutocomplete = ({onRefClick, showSearch, openTopic, openURL,
           return[];
         }
         try {
-        const d = await Sefaria.getName(inputValue);
+        let types = Sefaria.activeModule === Sefaria.SHEETS_MODULE ? ['Topic', 'User', 'Collection'] : undefined;
```

Comment:
why not empty array?

```suggestion
        let types = Sefaria.activeModule === Sefaria.SHEETS_MODULE ? ['Topic', 'User', 'Collection'] : [];
```